src/ucp/wireup/address.c

```diff
@@ -72,12 +72,18 @@ static size_t ucp_address_string_packed_size(const char *s)
     return strlen(s) + 1;
 }
 
-static int ucp_address_iface_attr_size(ucp_worker_t *worker)
+static size_t ucp_address_iface_attr_size(ucp_worker_t *worker)
 {
     return worker->context->config.ext.unified_mode ?
            sizeof(ucp_rsc_index_t) : sizeof(ucp_address_packed_iface_attr_t);
 }
 
+static int ucp_worker_iface_can_connect(uct_iface_attr_t *attrs)
+{
+    return (attrs->cap.flags &
```

Comment:
why need ?: operator
if really need to return 0/1 (which is not the case here BTW) can do !!